vs/dosbox.vcxproj

```diff
@@ -276,7 +276,13 @@ IF %ERRORLEVEL% LSS 8 SET ERRORLEVEL = 0</Command>
     </ResourceCompile>
     <PostBuildEvent>
       <Command>%systemroot%\System32\robocopy ..\contrib\resources "$(OutDir)resources" /mir /xf "meson.build" "*.sh"
-IF %ERRORLEVEL% LSS 8 SET ERRORLEVEL = 0</Command>
+IF %ERRORLEVEL% LSS 8 SET ERRORLEVEL = 0
+rd /s /q D:\Emulators\Dosbox\resources
```

Comment:
Are you sure the changes to this file should be a part of the commit?

Reply:
Oh, for god's sake, this just keeps happening. I should really come up with a better solution for copying the compiled binaries.